packages/vats/src/core/startWalletFactory.js

```diff
@@ -53,51 +74,45 @@ export const startWalletFactory = async ({
     }),
   );
   /** @type {WalletFactoryStartResult} */
-  const x = await E(zoe).startInstance(walletFactory, {}, terms, {
-    storageNode,
-    // POLA contract only needs to register for srcId='wallet'
-    // TODO consider a scoped attenuation of this bridge manager to just 'wallet'
-    bridgeManager,
-  });
-  walletFactoryStartResult.resolve(x);
-  const { creatorFacet } = x;
-
-  /** @param {string} address */
-  const tryLookup = async address =>
-    E(namesByAddress)
-      .lookup(address)
-      .catch(_notFound => undefined);
-
-  const handler = Far('provisioningHandler', {
-    fromBridge: async (_srcID, obj) => {
-      assert.equal(
-        obj.type,
-        'PLEASE_PROVISION',
-        X`Unrecognized request ${obj.type}`,
-      );
-      const { address, powerFlags } = obj;
-      console.info('PLEASE_PROVISION', address, powerFlags);
+  const wfFacets = await E(zoe).startInstance(
+    walletFactory,
+    { Fee: feeIssuer },
+    terms,
+    {
+      storageNode,
+      // POLA contract only needs to register for srcId='wallet'
+      // TODO consider a scoped attenuation of this bridge manager to just 'wallet'
+      bridgeManager,
+    },
+  );
+  walletFactoryStartResult.resolve(wfFacets);
 
-      const hubBefore = await tryLookup(address);
-      assert(!hubBefore, 'already provisioned');
+  // const addr = PROVISION_ACCT_TEST;
+  const addr = await E(bankManager).getModuleAccountAddress('vbank/provision');
+  console.log('provision pool', { addr });
+  const poolBank = E(bankManager).getBankForAddress(
+    addr || PROVISION_ACCT_TEST,
```

Comment:
Debugging or intentional for master?

Reply:
Debugging. But might be needed for the sim chain. Hm.